mungegithub/mungers/approval-handler.go

```diff
@@ -127,7 +149,7 @@ func (h *ApprovalHandler) Munge(obj *github.MungeObject) {
 		obj.WriteComment(*newMessage)
 	}
 
-	if !approversHandler.IsApproved() {
+	if !approversHandler.IsApprovedExtended() {
```

Comment:
Perhaps `IsApprovedWithIssueChecked`? The function checks whether the PR has an issue or has `/approve no-issue`.